### PR TITLE
Fix Japscan

### PR DESCRIPTION
**Pull request by az4521**
uses a *new* way to scrape websites, background webview screenshotting. this could potentially be used to scrape other annoying sites like mangago.

### CODE REVIEW

File: src/fr/japscan/src/eu/kanade/tachiyomi/extension/fr/japscan/Japscan.kt [hunk 1]
```diff
@@ -46,42 +55,72 @@ class Japscan : ParsedHttpSource() {
...
+        val cleanupjs = "var db=document.body,chl=db.children;for(db.appendChild(document.getElementsByTagName('CNV-VV')[0]);'CNV-VV'!=chl[0].tagName;)db.removeChild(chl[0]);window.variable={w:chl[0].all_canvas[0].width,h:chl[0].all_canvas[0].height};"
         val request = chain.request()
         val url = request.url().toString()
 
         val newRequest = request.newBuilder()
             .url(url.substringBefore(indicator))
             .build()
         val response = chain.proceed(newRequest)
-
+        Log.d("japscan", "network req $url")
```
Comment (happywillow0) [inline src/fr/japscan/src/eu/kanade/tachiyomi/extension/fr/japscan/Japscan.kt:69]: FYI, you left a debug log here.

**Comment by Bamaraj09** [comment]
what happened ??

**Comment by AbdullahM0hamed** [comment]
We call this an illusion.

**Comment by az4521** [comment]
> what happened ??

they took Alsace-Lorraine

**Comment by Bamaraj09** [comment]
I literally poped off when i see on github that japscan going to be fixed... rip

**Comment by az4521** [comment]
this extension was commissioned and the client refused payment. it works perfectly. it won't be merged unless payment is received.

**Comment by Bamaraj09** [comment]
I see how much $ was the price ?

**Comment by az4521** [comment]
300CAD, "as a joke", it was revealed after the extension worked

**Comment by Bamaraj09** [comment]
300 $ LMAOOO

**Comment by az4521** [comment]
ikr? why do you think i agreed to make it

**Comment by Bamaraj09** [comment]
For 300 $ i re-code the entire application lol

**Comment by az4521** [comment]
for 300$ i would deal with a supremely annoying french man and a website with JS that looks like it came directly from a hex editor

**Pull request closed without merging**